pyproject.toml

```diff
@@ -15,6 +15,7 @@ python = ">= 3.8, < 3.12"
 httpx = "^0.23.0"
 pydantic = "^1.9.2"
 tqdm = "^4.60.0"
+click = "8.1.8"
```

Comment:
Why is there a new library?

Reply:
it is not new, it is just fixed to a version that does not have this update,
where a function requires a ctx parameter. 
See how the previous build failed:
https://github.com/brighter-ai/redact-client/actions/runs/14957417642/job/42014963970